tests/containers/push_container_image_to_acr.pm

```diff
@@ -9,7 +9,7 @@
 
 use Mojo::Base 'publiccloud::basetest';
 use testapi;
-use containers::urls qw(get_suse_container_urls get_urls_from_var);
+use containers::urls qw(get_suse_container_urls get_container_test_image);
```

Comment:
What is the difference between `get_suse_container_urls` and `get_container_test_image` that's pretty confusing and not very descriptive.